public/css/main.scss

```diff
@@ -1,3 +1,118 @@
+@mixin type($background-color) {
```

Comment:
can you copy this to its own .scss file and import it? (put the file in the same directory)